util/topsql/state/state.go

```diff
@@ -27,7 +27,7 @@ const (
 
 // GlobalState is the global Top-SQL state.
 var GlobalState = State{
-	Enable:                atomic.NewBool(DefTiDBTopSQLEnable),
+	enable:                atomic.NewBool(DefTiDBTopSQLEnable),
```

Comment:
Why does the state have a default? I think it should be always false initially, in all cases.